src/test/java/com/icthh/xm/ms/mstemplate/config/IntegrationTestConfiguration.java

```diff
@@ -13,14 +19,10 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Primary;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClientService;
```

Comment:
do we really need to import that bean in ms-template?
in case we made feign commons as optional i think all tests should be in commons as well. 
so finally ms template should be able to **run tests** and **start** without error in next cases:
1. feign commons added to gradle build
2. feign commons removed from gradle build